### ISSUE TITLE
I want to connect two networks,but No gradients provided for any variable

### ISSUE THREAD

**Issue by Eify666666** [tensorflow/tensorflow#30527]
I trained a network, and I want to add another network before the trained network

`with tf.name_scope('inNN'):
    T_lamb = tf.placeholder(tf.float32, [None, 2], name='inNN_inputs')
    Ts = tf.placeholder(tf.float32, [None, 1], name='T_labels')
    Ds = tf.placeholder(tf.float32, [None, 4], name='D_labels')
    DLs = tf.placeholder(tf.float32, [None, 5], name='D_lamb')

weight, bias = get_w_b()
D_logits_op = inNN(T_lamb, None)
prediction = fwNN(DLs, None, weight, bias)
with tf.variable_scope('T_loss'):
    T_loss_op = losses(prediction, Ts)
with tf.variable_scope('D_loss'):
    D_loss_op = losses(D_logits_op, Ds)
global_step = tf.Variable(0, trainable=0)

valid_op = evaluation(prediction, Ts)
with tf.variable_scope('train_step'):
    train_step_op = training(T_loss_op, LEARNING_RATE, global_step)  # error: No gradients provided for any variable, check your graph for ops that do not support gradients`

when I choose the D_loss_op in the training. The network works

**Comment by ravikyram** [comment]
This question is better asked on StackOverflow since it is not a bug or feature request. There is also a larger community that reads questions there and provide better and faster support for such issues. Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
